pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -1,58 +1,396 @@
 <section class="my-4">
+  {{ if .Envelope }}
   <section>
     <div class="mb-4 md:mb-0">
-      <span class="font-semibold">Transaction ID: {{ .ID }}</span>
+      <span class="font-semibold">Transaction ID: {{ .Envelope.Transaction.Txid }}</span>
```

Comment:
To display the ivms101 data in the form, I mostly followed what was done in the `TransactionPreview` partial [here](https://github.com/trisacrypto/envoy/blob/5ea071af05315de9b3f12b7e7d548e710fe80c62/pkg/web/templates/partials/transaction/transaction_preview.html#L39).